crates/subspace-core-primitives/src/objects.rs

```diff
@@ -156,3 +158,12 @@ impl GlobalObject {
         }
     }
 }
+
+/// Mapping of objects stored in the history of the blockchain
+#[derive(Debug, Default, Clone, PartialEq, Eq, Hash, Encode, Decode, TypeInfo)]
```

Comment:
I also added Ord here, so we can order mappings in archive order. (At least in the way we’re producing them right now.)